src/docx/assets/zarzadzenie/word/document.xml

```diff
@@ -0,0 +1,150 @@
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="centerAlignedTitleGreenColor"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Poradnia Psychologiczno – Pedagogiczna we Wrześni</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="100"/>
+        <w:pStyle w:val="Uppercase"/>
+      </w:pPr>
+      <w:r>
+        <w:t>PPP</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="City-and-date"/>
+      </w:pPr>
+      <w:r>
+        <w:t>{{ city }}, {{ date }}</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="500"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Zarządzenie nr {{ no }}/2020</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>dyrektora Poradni z dnia {{ date }} r.</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>w sprawie powołania zespołu orzekającego</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>§1</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="normalParagraph"/>
+      </w:pPr>
+      <w:r>
+        <w:t xml:space="preserve">Dyrektor Poradni Psychologicznej w {{ supportCenter.city }}, {{ supportCenter.post }}, {{ supportCenter.address }}, na podstawie § 4 ust. 1-6 rozporządzenia Ministra Edukacji Narodowej z dnia 7 września 2017 r. w sprawie orzeczeń i opinii wydawanych przez zespoły orzekające działające w publicznych poradniach psychologiczno – pedagogicznych, (Dz. U. z 2017 r. poz. 1743) powołuje zespół orzekający do rozpatrzenia wniosku </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ applicant.name }} </w:t>
+      </w:r>
+      <w:r>
+        <w:t xml:space="preserve">zam. </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ applicant.street }}/{{ applicant.houseNumber }}, {{ applicant.postalCode }} {{ applicant.city }} </w:t>
+      </w:r>
+      <w:r>
+        <w:t xml:space="preserve">w sprawie wydania orzeczenia o potrzebie indywidualnego obowiązkowego rocznego przygotowania przedszkolnego dla </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ child.name }} </w:t>
+      </w:r>
+      <w:r>
+        <w:t xml:space="preserve">ur. </w:t>
+      </w:r>
+      <w:r>
+        <w:rPr>
+          <w:b/>
+        </w:rPr>
+        <w:t xml:space="preserve">{{ child.birthDate }} r. {{ child.birthPlace }}.</w:t>
```

Comment:
Can we join that separate element into single text run?
```suggestion
        <w:t>{{ child.name }}, ur.  {{ child.birthDate }} r. {{ child.birthPlace }}.</w:t>
```

Reply:
I had to separate these elements, because _ur._ text has normal font weight and the rest of the sentence like `{{ child.name }}` and `{{ child.birthDate }} r. {{ child.birthPlace }}.` are bolded. <br>
It looks as follow in the original template - **Wiktor Rzeźniczak** ur. **12.12.1912 r. Pachy Wielkie.**